eng/pipelines/client.yml

```diff
@@ -117,7 +117,7 @@ jobs:
 
       - script: |
           git clone https://github.com/JonathanGiles/DependencyChecker.git
-          mkdir input && cp dependencies.json input/
+          mkdir input && cp eng/DependencyChecker/client_dependencies.json input/dependencies.json
```

Comment:
Does the relative paths to the pomUrls in the file also need to be updated?

Reply:
The contents of the dependency file do not need to change, I just changed where it's being copied from. It's still going to be copied into the same location that dependency checker needs it to be.

Reply:
Now that we have the ```java - aggregate-reports``` pipeline, we should probably make this change there as ```client.yml``` is nearly 100% obsolete. Just need to look at moving a repo wide code coverage collection step across.

Reply:
@mitchdenny, I don't think you saw the second commit (which was in about 30 minutes before your comment). I'd changed it in both places and run the aggregate-reports pipeline to verify.